packages/rn-tester/js/examples/Text/TextExample.android.js

```diff
@@ -798,6 +798,12 @@ class TextExample extends React.Component<{...}> {
           <Text style={{textTransform: 'capitalize'}}>
             This text should be CAPITALIZED.
           </Text>
+          <Text>
+            Capitalize a date:
+            <Text style={{textTransform: 'capitalize'}}>
+              the 9th of november, 1998
+            </Text>
+          </Text>
```

Comment:
let's add one for 2 digit dates as well

Reply:
Done! 
It still works:
![image](https://user-images.githubusercontent.com/40902940/146684794-666903df-51d8-411b-949d-859a266e5fca.png)